src/main/java/com/google/sps/data/VideoState.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+public enum VideoState {
```

Comment:
nit: don't you want to have a video class having both video state and url? Video state enum can be defined in the nested class.

Reply:
That's the roomstate class. I defined it here just to have it available without referencing another class.

Reply:
It's up to you to define in a separate file or nested in room class. Just a clarification that a nested class can still be accessible by other classes.